code/style.css

```diff
@@ -1,7 +1,414 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+body {
```

Comment:
Some fixes need to be done regarding responsiveness:
<img width="387" alt="Skärmavbild 2021-02-09 kl  14 23 03" src="https://user-images.githubusercontent.com/51529201/107369649-53710100-6ae2-11eb-8030-19a70afc24fa.png">